ddl/ddl_api.go

```diff
@@ -4932,6 +4923,25 @@ func (d *ddl) AlterTableSetTiFlashReplica(ctx sessionctx.Context, ident ast.Iden
 	return errors.Trace(err)
 }
 
+func checkTiFlashSupportable(schema *model.DBInfo, tb table.Table) error {
```

Comment:
Maybe this is a better name:

```suggestion
func isTableTiFlashSupported(schema *model.DBInfo, tb table.Table) error {
```